tests/implementation/warnings/warnings.test.ts

```diff
@@ -71,3 +71,13 @@ describe('Delete Warnings', () => {
     await expect(deleteWarning('abcde')).rejects.toThrow();
   });
 });
+describe('Delete Guild Warnings', () => {
+  test('Valid', async () => {
+    await createWarning(1, 2, 'boring reason', 1);
+    await createWarning(1, 2, 'another boring reason', 1);
+    await expect(deleteGuildWarnings(2)).resolves.not.toThrow();
```

Comment:
Maybe add an extra expect to ensure that it has actually been deleted from the database?

Reply:
Can do, need to wait for #68